### DISCUSSION TITLE
Diablo I issue with character experience

### DISCUSSION THREAD

**Discussion by Mett0ttim** [diasurgical/DevilutionX#7581] [Q&A] [answered]
I am playing a Rouge playing on Normal difficultly and got to Hell before I had to restart a new game. I hit some monster I couldn't beat, so I started a new game to gain experience, but I have played 13 lvls without gain one point. I am searching for any information or solutions. I am a not a tech person, so go easy on me and I'll do my best to answer follow ups.

**Comment by StephenCWills** [comment, marked as answer]
You won't gain any experience from monsters that are ten levels below you. Though, I doubt that you made it all the way to dlvl 13 without gaining even a single experience point. Are you absolutely certain that the monsters on dlvl 13 don't give you any experience?

**Reply by julealgon** [reply]
I assume he meant he didn't get any level ups, not any experience whatsoever. I could be wrong though.

**Comment by AJenbo** [comment]
what level are you?

**Reply by Mett0ttim** [reply]
I am lvl 21 now, but was closer to lvl 15 when I posted this.

**Comment by julealgon** [comment]
I feel like this is just the natural imbalance of the original Diablo. These difficulties were never balanced properly and XP gain is an absolute mess.

This might seem like a dumb answer, but you are just wasting your time by playing this game on anything other than the normal experience. It was not designed to be played past that for the most part.

Having said that, if you _do_ want experience, don't play lower dungeons: just do hell/crypt (in hellfire) runs and such. Unless you are just after amassing money, in which case lower dungeons (in nightmare and up) can be of some help.

Again, if you want anything resembling an "endgame experience" in Diablo, I'd look elsewhere, like one of the various mods that exist for the game.

**Reply by AJenbo** [reply]
It appears they where playing on Normal difficultly, they reached Hell as in level 13-16, not that they switched to Hell difficultly

**Reply by julealgon** [reply]
You're right, I misread that. Oops 😅

**Comment by Mett0ttim** [comment]
Thank you all for giving me your input. It appears my character wasn't close to the monsters I was killing. Things changed once got closer. I jumped to conclusions. Thank you all for answering my question.
